src/_posts/platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md

```diff
@@ -129,7 +129,7 @@ platform/app/2000-01-01-review-apps %})) we suggest to modify the tasks related
 
 ### How Precise Is Scalingo Scheduler?
 
-Execution time can be delayed by a few minutes. Indeed in order to execute tasks on an application we need to pull the application image which could take a few minutes depending on its size.
+Execution time can be delayed by a maximum of 10% of the interval between 2 tasks (up to 30 minutes). This is to prevent load spikes on the infrastructure and to give time for an application pull its image.
```

Comment:
```suggestion
Execution time can be delayed by a maximum of 10% of the interval between 2 tasks (up to 20 minutes).

* `*/10 * * * *` (every 10 minutes): task will be executed every 10 to 11 minutes
* `0 */1 * * *` (every hour): task will be executed between minute 0 and minute 6 of each hour
* `0 0 * * *`: (everyday at midnight): task will be executed between 00:00 and 00:20 UTC

This is to prevent load spikes on the infrastructure and to give time for our infrastructure to pull your application images.
```